packages/dmn-js/test/spec/NavigatedViewerSpec.js

```diff
@@ -103,7 +113,12 @@ describe('NavigatedViewer', function() {
       // can open decisions
       expect(drdView.element.$instanceOf('dmn:Definitions')).to.be.true;
 
-      editor.open(drdView, done);
+      editor.open(drdView)
+        .then(
+          result => done(result.warnings[0]))
```

Comment:
Maybe add a note that we have to remove the `done` here as well. But we will probably remove all of them once the migration is done.

Reply:
Yep - I plan to migrate to `async` tests once I have the key APIs promisfied. I think I would have to add many comments, so I would propose to not do that.

I already put a comment in the commit, that migrating away from `done` in the tests is planned as part of this promisification endavour.